README.md

```diff
@@ -125,4 +125,4 @@ Feel free to check the [issues page](https://github.com/Olivier-Kango/vet_clinic
 This project is [MIT](./LICENSE.txt) licensed.
 
 
```

Comment:
- [ ] `queries.sql` file: Kindly make sure all queries that answer the below questions are added to the `queries.sql` file and all of them should be correct. Right now, you only add screenshots and missed their code for the different specific files.
  
![Screenshot from 2022-12-15 20-30-23](https://user-images.githubusercontent.com/83042742/207940515-9a1f5832-df55-4450-adef-06a31247fdc6.png)